resources/XZ_Supply_chain_attack.md

```diff
@@ -0,0 +1,108 @@
+
+On further investigation of the person responsible for the attack, who goes by the alias ‘Jia Tan’, it seems that they showed a pattern of introducing compromising open source contributions into key projects. This person, or very likely a group of people with considerable resources, has spent years gaining the trust of key open source developers and projects. A key and most concerning example of a malicious contribution that ‘Jia Tan’ made was to the libarchive repository to change instances of a safe call to a function to an unsafe version of that function (printf). This persona has not shown activity since the attack was discovered.
+
+## What does this mean for CMS systems?
+
+This attack should not affect us in any major way. Any crucial systems that CMS is responsible for should not have been running experimental versions of Debian or Fedora using systemd services, and that is the only way key systems would be affected. Systems that do not use Systemd services, such as MacOS, would not be affected. Systemd is a system processing monitor and suite of tools that is only available for the Linux operating system.
+
+Going forward, CMS should encourage the adoption of healthy standards such as:
+
+- Code review
+- Applying security scanning tools
+- Isolation of build environments
+- Ensuring that crucial systems maintained by contractors show compliance
+
+This is part of CISA’s secure by design initiative that is extremely relevant to this attack. Additionally, it is important to encourage the continued adoption of the CMS OIT Cybersecurity initiatives such as CMS (GRC) Governance Risk and Compliance.
+
+## What does this mean for open source at CMS?
+
+As an Open Source Program Office, it is largely our responsibility to foster practices that make attacks like this as difficult as possible. A key detail that allowed this attack to happen was an important open source project being poorly maintained. The xz project had only two people working on it and one of them turned out to be malicious. As an OSPO, it is important to establish practices that enforce swift detection and action towards projects that are poorly maintained. Additionally, the malicious actor also was allowed to alter the SECURITY.md file of the xz project to redirect to their personal email address. This is never something that should be allowed without review, and key stakeholders should be alerted if important files like the LICENSE or SECURITY.md are changed. Automated checks could have been used to raise suspicion sooner, such as linting tools to detect some of the more obviously malicious changes to important projects.
```

Comment:
IDEA: Link here to repo-scaffolder workflow that checks for changes to SECURITY.md